assets/app/view/game/route_selector.rb

```diff
@@ -149,16 +158,29 @@ def render
             ]),
             h(:tbody, trains),
           ]),
-          actions,
+          actions(render_halts),
         ])
       end
 
+      def halt_actions(route, revenue, subsidy)
```

Comment:
would it be easier to just pay out max halts instead of having this action?

Reply:
I feel it's a pretty important tactical decision about running for revenue or to help the company. The rules specifically call this out as a choice the director makes.

Reply:
hm i thought the choice was whether or not to use the halt or not. is there another choice? i don't think halts pay out, they only ever help the company. the choice would be to use the halt or not, you'd only not want to use it in very rare situations

Reply:
In a real life game with a M+N train, you decide how the +N stops are split between towns and halts. Since you can choose between upgrading to another halt or to a town, presumably you'd want the choice to get the revenue from the town if they were available.

Reply:
How strongly do you dislike how I have it now?

Reply:
i'll look at it some more this weekend